js/views/chatHeadsVw.js

```diff
@@ -18,6 +18,18 @@ module.exports = baseVw.extend({
         this.createChatHead(md).render().el
       );
     });
+
+    this.$chatHeadsContainer = $('.chatConversationHeads');
+    this.$headContainer = $('<div />');
+
+    this.showPerScroll = 10;
+    this.nextChatToShow = 0;
+
+    //listen to scrolling on container
+    this.scrollHandler = __.bind(
+        __.throttle(this.onScroll, 100), this
+    );
+    this.$chatHeadsContainer.on('scroll', this.scrollHandler);
```

Comment:
Since we're binding this to an element outside of our view, it needs to be unbound in the view's remove method. I know that now we are never removing the chat app, but in case we have a use case for that in the future, the view should do it's necessary cleanup.